src/ignitus-StudentSignUp/Components/Signup.js

```diff
@@ -20,32 +20,34 @@ class Signup extends React.Component {
 
   render() {
     return (
-      <div className="container _container-custom p-5">
-        <div className="row shadow">
-          <div className="col-md-6 p-0 container-image">
-            <img className="img-fluid _img-login d-block" src={loginImg} alt="login" />
-            <div className="text-below-image text-center">
-              <p className="mb-5 ">Let's get started</p>
-              <p>Skyrocket your career with best global opportunities</p>
-              <p>
-                <a href="/signup" className="text-center _link">
-                  {' '}
-                  I am a Professor
-                </a>
-              </p>
-            </div>
-          </div>
-          <div className="col-md-6  _container-form ">
-            <div className="my-4">
-              <img className="img-fluid _img mx-auto d-block" src={logo} alt="logo" />
+        <Fragment>
+        <div className="container _container-custom p-5">
```

Comment:
The Fragment is needed because it is the outermost container for the markup. An outer div would also work but a Fragment serves the same purpose without cluttering the dom (or virtual dom) with an unnecessary div

Reply:
i am sry if i don't understand but in this case i thought that `<div className="container _container-custom p-5">` is already the outermost container and the fragment is only needed as outer container if you have  a list of children